lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_node_lowering.cpp

```diff
@@ -58,8 +60,33 @@ class TSNodeLowering : public NodeLowering {
       return false;
     }
     LTC_CHECK_EQ(node->num_outputs(), ops.size());
-    for (size_t i = 0; i < ops.size(); ++i) {
-      loctx()->AssignOutputOp(ir::Output(node, i), ops[i]);
+    if (lazy_tensors::sys_util::GetEnvBool("LTC_IR_TS_SHAPE_TYPES", false)) {
```

Comment:
not sure if putting behind env is good or bad

Reply:
It's probably okay for now, although a python switch like this one: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/python/init.cpp#L608

would seem cleaner and more flexible.

Reply:
nitpick: do you need to prefix `sys_util` if you already included `using namespace lazy_tensors` ?

Reply:
re the python switch: I think that should go in `LoweringContext` but i'm not sure since there's no such flags being passedyet.
re `sys_util` yes the namespace resolution doesn't but actually I didn't need the `using`.